### DISCUSSION TITLE
求出 Apple Watch 版~

### DISCUSSION THREAD

**Discussion by Naco** [KyleBing/iphone#3] [Ideas]
由逼乎知道了作者的网站，然后找到了好用的一堆工具，也非常喜欢作者的出品风格，大赞~

就像提一个请求，系列对比工具能不能增加 Apple Watch 版？

**Comment by KyleBing** [comment]
有这打算，闲着的时候吧  😊

**Comment by KyleBing** [comment]
Apple Watch 版本已更

[https://kylebing.cn/tools/apple-watch/](https://kylebing.cn/tools/apple-watch/)

**Reply by Naco** [reply]
大赞~
